misk/src/main/kotlin/misk/web/jetty/JettyService.kt

```diff
@@ -337,15 +336,34 @@ class JettyService @Inject internal constructor(
         // the address is a null byte ('\0'). The address has no connection with filesystem
         // path names.
       }
-      server.destroy()
```

Comment:
turns out we weren't supposed to call destroy hehe oop, it just causes a meaningless error log for people